packages/expo-blur/jest.config.js

```diff
@@ -0,0 +1,11 @@
+const withEnzyme = require('jest-expo-enzyme');
+const { withWatchPlugins } = require('jest-expo/config');
+const withExpoModuleScripts = require('expo-module-scripts/createJestPreset');
+
+module.exports = withWatchPlugins({
+  projects: [
+    require('jest-expo/ios/jest-preset'),
+    require('jest-expo/web/jest-preset'),
+    // require('jest-expo/android/jest-preset'),
```

Comment:
What needs to happen to get Android working?

Reply:
oops